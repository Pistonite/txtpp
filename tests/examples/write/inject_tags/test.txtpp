The content of the write directive will not have tags injected,
because it is technically output of a directive.

0TXTPP#tag TAG_IN_WRITE
1TXTPP#run echo foo
3TXTPP#write some content with TAG_IN_WRITE in it.

However, since write is a directive, you can store its output in a tag

0TXTPP#tag TAG_WRITE
1TXTPP#write some content with TAG_IN_WRITE in it.
1
Note that the tag is still not injected, but now both TAG_IN_WRITE and TAG_WRITE have stored content.