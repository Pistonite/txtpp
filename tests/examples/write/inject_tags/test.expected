The content of the write directive will not have tags injected,
because it is technically output of a directive.

some content with TAG_IN_WRITE in it.
However, since write is a directive, you can store its output in a tag

Note that the tag is still not injected, but now both foo
 and some content with TAG_IN_WRITE in it.
 have stored content.
